### ISSUE TITLE
Images from dev.to are not displayed on discord

### ISSUE THREAD

**Issue by Mcdostone** [thepracticaldev/dev.to#309]
### Current Behavior
I wanted to share my joy to see [dev.to](dev.to) open sourced. I posted this link https://dev.to/ben/devto-is-now-open-source-5n1 on a discord developer channel (grafikart.fr For those who know) . The problem is no image is displayed in the thread.

I don't know if the issue comes from dev.to or the discord app but links from other websites (youtube for instance) are displayed properly on discord...

### Expected Behavior
When you post a link on discord, you should see the image attached to the dev.to article.

### Screenshots
![screenshot from 2018-08-09 13-25-59](https://user-images.githubusercontent.com/6984659/43915138-cc2b2c38-9bd7-11e8-82ed-909d10e56046.png)

### Additional Info
Ubuntu 18.10
Discord 0.0.5 (august 8, 2018)

**Comment by rhymes** [comment]
Mmm this may or may not be an issue with the app, Discord. The dev.to posts use [open graph](http://ogp.me/) to describe metadata, the one related to the image is `og:image` which, in the case of the post you linked is the following:

```html
<meta property="og:image" content="https://res.cloudinary.com/practicaldev/image/fetch/s--DXkV9vkl--/c_imagga_scale,f_auto,fl_progressive,h_500,q_auto,w_1000/https://thepracticaldev.s3.amazonaws.com/i/o6fs3rdrz51u0sn32efg.png" />
```

which renders the title image:

![devto opensource image](https://res.cloudinary.com/practicaldev/image/fetch/s--DXkV9vkl--/c_imagga_scale,f_auto,fl_progressive,h_500,q_auto,w_1000/https://thepracticaldev.s3.amazonaws.com/i/o6fs3rdrz51u0sn32efg.png)

Have you tried with other posts?

**Comment by Zhao-Andy** [comment]
Hey ya, @rhymes you’re on the right track with the `<meta>` tag. I spent some time on this a few weeks back since I use Discord and it bothered me that our images don’t show up. :upside_down_face: 

I think the conclusion that I came to was that Cloudinary sends a `401 Access Denied` response when Discord’s image servers tries to fetch our images. Whoever takes this up, feel free to follow where I left off if you’d like.
